ncm-ssh/src/main/pan/components/ssh/schema.pan

```diff
@@ -48,7 +49,8 @@ type ssh_daemon_options_type = {
     "ClientAliveInterval"               ? long
     "DenyGroups"                        ? string
     "DenyUsers"                         ? string
-    "HostKey"                           ? string
+    "GSSAPIStrictAcceptorCheck"         ? ssh_yesnostring
+    "HostKey"                           ? string[]
```

Comment:
for multiple `Hostkey` you could use a 
```pan
type sshd_hostkeys = {
    'rsa' ? string
    'dsa' ? string
...
};
```
and use a new attribute for the multiline
```pan
Hostkeys ? sshd_hostkeys
```
 so as key the protocol and value the filename. the key does not appear in the config file, but that's what makes it multiple. it also makes no sesne to have more keys than supported protocols, or more than one file for the same protocol

for match, check the https://github.com/quattor/configuration-modules-core/blob/master/ncm-metaconfig/src/main/metaconfig/ssh/pan/schema.pan